src/main/resources/initialize.sql

```diff
@@ -0,0 +1,447 @@
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."Mentor_Availability"
+    OWNER to postgres;
+
+
+--12 Table: public.MTT_InterestedIn
+
+-- DROP TABLE IF EXISTS public."MTT_InterestedIn";
+
+CREATE TABLE IF NOT EXISTS public."MTT_InterestedIn"
+(
+    "MenteeID" integer NOT NULL DEFAULT nextval('"MTT_InterestedIn_MenteeID_seq"'::regclass),
+    "TopicsID" integer NOT NULL DEFAULT nextval('"MTT_InterestedIn_TopicsID_seq"'::regclass),
+    CONSTRAINT "MTT_InterestedIn_pkey" PRIMARY KEY ("MenteeID", "TopicsID"),
+    CONSTRAINT "MTT_InterestedIn_MID_fkey" FOREIGN KEY ("MenteeID")
+        REFERENCES public."Mentee" ("MenteeID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION,
+    CONSTRAINT "MTT_InterestedIn_TID_fkey" FOREIGN KEY ("TopicsID")
+        REFERENCES public."Topics" ("TopicsID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."MTT_InterestedIn"
+    OWNER to postgres;
+
+
+--13 Table: public.ODD_Makes
+
+-- DROP TABLE IF EXISTS public."ODD_Makes";
+
+CREATE TABLE IF NOT EXISTS public."ODD_Makes"
+(
+    "OnlineDonorID" integer NOT NULL DEFAULT nextval('"ODD_Makes_OnlineDonorID_seq"'::regclass),
+    "DonationID" integer NOT NULL DEFAULT nextval('"ODD_Makes_DonationID_seq"'::regclass),
+    "Date" timestamp with time zone NOT NULL,
+    CONSTRAINT "ODD_Makes_pkey" PRIMARY KEY ("OnlineDonorID", "DonationID"),
+    CONSTRAINT "ODD_Makes_DID_fkey" FOREIGN KEY ("DonationID")
+        REFERENCES public."OnlineDonation" ("DonationID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION,
+    CONSTRAINT "ODD_Makes_ODID_fkey" FOREIGN KEY ("OnlineDonorID")
+        REFERENCES public."OnlineDonor" ("OnlineDonorID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."ODD_Makes"
+    OWNER to postgres;
+
+
+--14 Table: public.AS_Makes
+
+-- DROP TABLE IF EXISTS public."AS_Makes";
+
+CREATE TABLE IF NOT EXISTS public."AS_Makes"
+(
+    "AdminID" integer NOT NULL DEFAULT nextval('"AS_Makes_AdminID_seq"'::regclass),
+    "SessionID" bigint NOT NULL DEFAULT nextval('"AS_Makes_SessionID_seq"'::regclass),
+    CONSTRAINT "AS_Makes_pkey" PRIMARY KEY ("AdminID", "SessionID"),
+    CONSTRAINT "AS_Makes_AID_fkey" FOREIGN KEY ("AdminID")
+        REFERENCES public."Admin" ("AdminID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION,
+    CONSTRAINT "AS_Makes_SID_fkey" FOREIGN KEY ("SessionID")
+        REFERENCES public."Session" ("SessionID") MATCH SIMPLE
+        ON UPDATE NO ACTION
+        ON DELETE NO ACTION
+)
+
+TABLESPACE pg_default;
+
+ALTER TABLE IF EXISTS public."AS_Makes"
+    OWNER to postgres;
+
+
+--15 Table: public.AM_Assigns
+
+-- DROP TABLE IF EXISTS public."AM_Assigns";
+
+CREATE TABLE IF NOT EXISTS public."AM_Assigns"
```

Comment:
Please remove this table because we don't need to track which admin assigned which mentor. This was a database mistake. If and admin assigns a mentor, it should just be added to the mentor_gives_sessions relation.